pkg/systemd/overview-cards/serverTime.js

```diff
@@ -61,13 +62,10 @@ export function ServerTime() {
     });
 
     self.format = function format(and_time) {
-        var string = self.utc_fake_now.toISOString();
-        if (!and_time)
-            return string.split('T')[0];
-        var pos = string.lastIndexOf(':');
-        if (pos !== -1)
-            string = string.substring(0, pos);
-        return string.replace('T', ' ');
+        moment.locale(cockpit.language); // HACK: when done at the top of the file, c.language is undefined
```

Comment:
I think it should rather live in pkg/systemd/overview.jsx . @allisonkarlitskaya needs that for PR #13473 as well.